docs/pywbemcli/cmdshelp.rst

```diff
@@ -185,14 +185,14 @@ Help text for ``pywbemcli class`` (see :ref:`class command group`):
       -h, --help  Show this message and exit.
 
     Commands:
-      associators   List the classes associated with a class.
-      get           Get a class.
       enumerate     List top classes or subclasses of a class in a namespace.
-      tree          Show the subclass or superclass hierarchy for a class.
-      references    List the classes referencing a class.
+      get           Get a class.
+      delete        Delete a class.
       invokemethod  Invoke a method on a class.
+      references    List the classes referencing a class.
+      associators   List the classes associated with a class.
       find          List the classes with matching class names on the server.
-      delete        Delete a class.
+      tree          Show the subclass or superclass hierarchy for a class.
```

Comment:
The changes in `docs/pywbemcli/cmdshelp.rst` are unrelated to the purpose of this PR. I suggest moving them out of this PR.

Reply:
This change is driven by pr #450, which added a method that orders commands and subcommands by their order in the source file.  It was approved and merged 5 Nov 2019.  Somewhere we apparently did not rebuild the doc file so this pr catches this change.

Reply:
ok.